Hallo.AspNetCore.Mvc/HalJsonOutputFormatter.cs

```diff
@@ -56,27 +59,17 @@ public HalJsonOutputFormatter(JsonSerializerOptions serializerOptions)
 
         public override async Task WriteAsync(OutputFormatterWriteContext context)
         {
-            var representationGenerator = GetRepresentationGenerator(context.HttpContext.RequestServices, context.ObjectType);
-            if (representationGenerator == null)
+            var json = await HalJsonGenerator.GenerateHalJson(context.HttpContext, context.Object, SerializerOptions);
```

Comment:
This now calls on the aspnetcore package which generates the hal json which is returned below otherwise it falls back on the base output formatter to serialize the object into non hal json